Codeinwp/neve
# Single post options

### Summary
- Added the single post options according to the [design](https://www.figma.com/file/LBW82Rm2SA5rvvLdKrRnBl/Neve-Core-UI?node-id=296%3A7887). Some controls were changed during the implementation based on what I discussed with John and Marius
- Besides the new options for the single post, the controls for Blog meta that are in Pro, were moved in lite.

### Will affect visual aspect of the product
NO

### Test instructions
- There are some controls that are added in Pro. Check https://github.com/Codeinwp/neve-pro-addon/pull/1374
- First of all, you need to change `NEVE_VERSION` variable to be `3.0.0`. There is a check in Neve Pro that verify that variable and if it's lower than 3.0, the blog meta will be added from the pro, else they will be added from lite 
- Check that all the options are working as they should be
- Check the meta options for Blog too.
- The meta options from the single post have their default values from blog options. If you don't change the single ones, the ones from the blog should apply.

@abaicus I've created a new CSS file as you've asked. It's named `single-new.scss`.
I also made some small modifications in `_blogpost-meta.scss` and in `_comments.scss`.
I modified the `_media-queries.scss` too so I can hook responsive classes.

Closes Codeinwp/neve-pro-addon#1110.
